src/mumble/GlobalShortcut_xdp.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2007-2022 Aleix Pol Gonzalez <aleixpol@kde.org>.
+// All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "GlobalShortcut_xdp.h"
+
+#include "Settings.h"
+#include "Global.h"
+#include "globalshortcuts_portal_interface.h"
+#include "portalsrequest_interface.h"
+#include "GlobalShortcutTypes.h"
+
+#include <QtGui/QWindow>
```

Comment:
```suggestion
#include <QWindow>
```
Including Qt stuff from "subdirectories" is considered bad practice and should generally not be done (you might end up including internal Qt headers that are not part of the public API)

Reply:
I am aware. I only did it this way because I saw several submodule includes in mumble's codebase. Removing.